vertx-opentracing/src/main/java/io/vertx/tracing/opentracing/OpenTracingContext.java

```diff
@@ -0,0 +1,48 @@
+package io.vertx.tracing.opentracing;
+
+import io.opentracing.Span;
+import io.vertx.core.Context;
+import io.vertx.core.Vertx;
+
+/**
+ * OpenTracingContext adds helpers for associating and disassociating spans with the current {@link
+ * Context}
+ */
+public class OpenTracingContext {
+
+  static final String ACTIVE_SPAN = "vertx.tracing.opentracing.span";
```

Comment:
why not private?

Reply:
I needed access to the string in the OpenTracingTracer. For why, see my comment below.